XenAdmin/TabPages/GeneralTabPage.cs

```diff
@@ -1777,7 +1777,7 @@ private string poolUpdateString(Predicate<Pool_update> predicate)
                 // As of Ely we use CheckHostPatchesRequiringReboot to get reboot messages for a host
                 foreach (Host host in xenObject.Connection.Cache.Hosts)
                 {
-                    warnings.AddRange(CheckHostPatchesRequiringReboot(host));
+                    warnings.AddRange(CheckHostUpdatesRequiringReboot(host));

```

Comment:
The comment above the foreach needs to be updated as well (it currently says "CheckHostPatchesRequiringReboot")